app/poi_api/poi.py

```diff
@@ -1,32 +1,67 @@
-from datetime import datetime
-from app import common
-from app.firebase import firestore_db
-from flask import jsonify
+from enum import Enum
+from typing import Dict
+from app.common import BadDataError
+from app.poi_api.errors import POINotFoundError
+import json
 
 
 class POI:
     def __init__(
         self,
-        _id,
-        name,
-        clasification,
-        hours_of_operation,
-        address,
-        poi_type,
-        location,
-        image_url,
+        _id: str,
+        name: str,
+        classification: str,
+        hours_of_operation: Dict[str, str],
+        address: str,
+        poi_type: str,
+        location: Dict[str, str],
+        image_url: str,
     ):
         self._id = _id
         self.name = name
-        self.classification = clasification
+        self.classification = classification
         self.hours_of_operation = hours_of_operation
         self.address = address
         self.poi_type = poi_type
         self.location = location
         self.image_url = image_url
 
-    def to_dict(self):
-        poi_suggestion_dict = {
+    @staticmethod
+    def from_dict(dict: Dict[str, str]) -> "POI":
+        """
+        Creates a new POI object from a Python Dictionary
+
+        Args:
+            dict: Dictionary of key-value pairs corresponding to the poi.
+
+        Returns:
+            POI: from specified data
+
+        Raises:
+            BadDataError: If required data is missing from the dictionary
+        """

```

Comment:
This comment style is the google one, we're going with something more succinct. See some of the functions in `events` for the format we're going for.